### ISSUE TITLE
Add bias twice in complexlinear

### ISSUE THREAD

**Issue by newoneincntk** [Chenest/Complex_networks#1]
def forward(self, input):
        # bs*in_features*2
        assert (input.shape[-1] == 2)
        assert (len(input.shape) == 3)
        input = torch.stack([self.fc_r(input[:, :, 0]) - self.fc_i(input[:, :, 1]) + self.bias[:, 0],
                            self.fc_r(input[:, :, 1]) + self.fc_i(input[:, :, 0]) + self.bias[:, 1]], dim=-1)
        return input + self.bias

Add bias twice in complexlinear? Is there something wrong? Or something I miss?

**Comment by Chenest** [comment]
Yes, you're rigth, thanks.
